content/en/docs/tasks/observability/distributed-tracing/lightstep/index.md

```diff
@@ -11,8 +11,8 @@ test: no
 
 {{< boilerplate telemetry-tracing-tips >}}
 
-This task shows you how to configure Istio to collect trace spans and send them to [Lightstep](https://lightstep.com).
-Lightstep lets you analyze 100% of unsampled transaction data from large scale production software to produce meaningful
+This task shows you how to configure Istio to collect trace spans and send them to [ServiceNow Cloud Observability](https://lightstep.com) (SNCO).
+SNCO lets you analyze 100% of unsampled transaction data from large scale production software to produce meaningful
```

Comment:
SNCO is not an approved external acronym. We have to spell out Cloud Observability. After the first instance of ServiceNow (in this case in the title), you can omit ServiceNow